learners-guild/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
@sj if you use `set +e` and `set +x` in your script you wont need as much echo logging and it will stop if any command fails